clients/cli/src/config.rs

```diff
@@ -48,9 +48,9 @@ fn signers_of(
     name: &str,
     wallet_manager: &mut Option<Rc<RemoteWalletManager>>,
 ) -> Result<Option<SignersOf>, Box<dyn std::error::Error>> {
-    if let Some(values) = matches.values_of(name) {
+    if let Some(values) = matches.try_get_many(name).ok().flatten() {
         let mut results = Vec::new();
-        for (i, value) in values.enumerate() {
+        for (i, value) in values.copied().enumerate() {
```

Comment:
```suggestion
        for (i, value) in values.enumerate() {
```
We are parsing for an owned `String` type, so we can remove the `copied()`.